### PR TITLE
Project 12: Movies

### PR DESCRIPTION
**Pull request by Thessan**
**Link to my deployed project:** [https://top-movies.netlify.app/](https://top-movies.netlify.app/)

**My reflections on how this week's project turned out:** Super fun project, I completed the blue level and I'm happy with how it turned out ✨ Learnt more about how to work with JS and React and it was fun to try React route.

**Things I'd like to have clarified or explained in more detail:** I'm stuck on how to not get the opacity to apply to the movie details when hovering each movie, but it's a small detail I'll figure out later on.

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
+      <Switch> 
+        <Route path="/" exact>
+          <MovieList />
+        </Route>
+
+        <Route path="/movie/:movieID" exact> {/* Route to MovieDetails */}
+          <MovieDetails />
+        </Route>
+      </Switch>
+    </BrowserRouter>
```
Comment (rebkova) [inline code/src/App.js:19]: Super easy to read, I like it!

File: code/src/MovieList.css [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+}
+
+.movieThumbnail {
+  width: 100%;
+  height: 100%;
+}
+
+.movieTitle1 {
+  display: none;
+}
```
Comment (rebkova) [inline code/src/MovieList.css:39]: Very clearly structured CSS clasess from top to the bottom of the page. Makes it so easy to follow. 😍

File: code/src/MovieList.css [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+.movieDetails1 {
+  display: flex;
+  position: absolute;
+  flex-direction: column;
+  justify-content: flex-end;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left:0;
```
Comment (rebkova) [inline code/src/MovieList.css:85]: #oddlysatisfying 🥰

**Review by rebkova — APPROVED** [review]
Very nicely written code and extra points for making the page responsive! ⭐ You could make small fixes to your indentation to make your code cleaner. Great job with writing the descriptive comments, Therese! It makes it so much easier to go throught the code. I wish more people would do that.

File: code/src/MovieDetails.css [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+.movieDetailsThumbnail {
+    width: 242px;
+    height: 343px;
+    margin: 50px 60px 0px 80px;
+}
+    
+.movieDetails2 {
+    max-width: 600px;
+    margin: 0 55px 35px 80px;
+}
```
[the remark this answers is not in the document]
Comment (Thessan) [reply code/src/MovieDetails.css:84]: I'm a bit unsure what to fix here?

File: code/src/MovieDetails.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+        fetch(`https://api.themoviedb.org/3/movie/${movieID}?api_key=b230af0ba5dd0959429a398c958e6e4e&language=en-US&page=1`)
+            .then((response) => response.json())
+            .then((json) => {
+                setMovieDetails(json);
+            })
+    }, [movieID]);
+
+    return (  
+        <div className="movieContainer" style={{backgroundImage: `url(https://image.tmdb.org/t/p/w1280${movieDetails.backdrop_path})`}} alt={`${movieDetails.title}`}>
+            <>
```
[the remark this answers is not in the document]
Comment (Thessan) [reply code/src/MovieDetails.js:21]: Good catch, missed that! 👏

File: code/src/MovieList.css [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+  width: 338px;
+  height: 508px;
+  padding-left: 15px;
+  opacity: 0;
+}
+  
+.movieDetails1:hover {
```
[the remark this answers is not in the document]
Comment (Thessan) [reply code/src/MovieList.css:92]: Thank you! 👏😃

File: code/src/MovieDetails.css (hunk 4 above)
[the remark this answers is not in the document]
Comment (Thessan) [reply code/src/MovieDetails.css]: I've changed this, it's however a strange that my VS displays red on all those lines now, like the indentation is wrong 🙈

File: code/src/MovieDetails.js [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+            .then((response) => response.json())
+            .then((json) => {
+                setMovieDetails(json);
+            })
+    }, [movieID]);
+
+    return (  
+        <div className="movieContainer"
+            style={{backgroundImage: `url(https://image.tmdb.org/t/p/w1280${movieDetails.backdrop_path})`}}
+            alt={`${movieDetails.title}`}>
```
Comment (EmOlsz) [inline code/src/MovieDetails.js:22]: or just `alt={movieDetails.title}`

File: code/src/MovieList.css [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+.movieThumbnail {
+  width: 100%;
+  height: 100%;
+}
+
+.movieTitle1 {
+  display: none;
+}
+
+.releaseDate {
```
Comment (EmOlsz) [inline code/src/MovieList.css:41]: class names should be written in `kebab-case`

**Pull request closed without merging**